hyper/cmd/jupyter.go

```diff
@@ -58,10 +63,20 @@ var jupyterStopCmd = &cobra.Command{
 	},
 }
 
+var jupyterCreateCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create EC2 instance to run a remote jupyter server",
+	Run: func(cmd *cobra.Command, args []string) {
+		remoteConfiguration := config.GetRemote(remoteProfile)
+		aws.StartServer(manifestPath, remoteConfiguration.EC2Configuration, ec2InstanceType, amiID)
+	},
+}
+
 func init() {
 	rootCmd.AddCommand(jupyterCmd)
 	jupyterCmd.AddCommand(jupyterListCmd)
 	jupyterCmd.AddCommand(jupyterStopCmd)
+	jupyterCmd.AddCommand(jupyterCreateCmd)
```

Comment:
Instead of adding a create sub command, we'll want this to be use the same start and stop commands that we use for firefly and local. If you check the `.hyperdrive.example` in this repo you can see that there is now an example of what a "remote_type" definition for ec2 would look like. 

For our different "remote" constructs, we'll basically want to implement the same commands and just specify `--remote=my-named-ec2-remote` since `--remote` is a top-level flag we want to aim to just allow the same commands work locally or on whatever other remote type we want to support.

You'll also want to just have the cmd call the service-level function instead of directly calling your client, and the service will essentially just check the configuration and call the appropriate client-level function